### PR TITLE
added download reoprt(txt,csv) test case for provider

### PR DESCRIPTION
**Pull request by prachiyadav**

### CODE REVIEW

**Comment by hawkularqe** [comment]
Test FAILed.<br>
BuildID: [167](http://jenkins.jonqe.lab.eng.bos.redhat.com:9080/job/miq-mw-ui-tests/167/)
<br> Gif records:<br>

**Comment by mattmahoneyrh** [comment]
Please consider adding the following two steps:

1) Delete the files prior to the Download (could be done in  @pytest.fixture

Ex:
file = "{}{}".format(os.getenv("HOME"),'/Downloads/Middleware*.*')
r = glob.glob(file)
for i in r:
    os.remove(i)


2) Assert that the file(s) exist after the download.

**Comment by mattmahoneyrh** [comment]
Python module 'glob' and 'fnmatch' will need to be added to requirements.txt prior to the merge, otherwise these tests will fail due to the modules not being present.

File: views/providers.py [hunk 1]
```diff
@@ -200,8 +200,7 @@ def delete_hawkular_provider(self):
 
         # Verify if the provider is deleted from the provider list.
 
-        assert ui_utils(self.web_session).waitForElementOnPage(By.XPATH, "//a[@title='{}']".format(
-            self.web_session.HAWKULAR_PROVIDER_NAME), 120, exist=False)
+
```
Comment (mattmahoneyrh) [inline views/providers.py]: I do not see why providers.py has this new assert - should not have any changes in providers.py. Please remove this file from the PR/Branch.

File: conf/properties.properties [hunk 2]
```diff
@@ -21,7 +21,7 @@ DB_PORT = 5432
 
 # Browser that will be used (needs to be installed and PATH set properly, see also README.md )
 # Firefox|Chrome
-BROWSER = Firefox
+BROWSER = Chrome
 
 BROWSER_WIDTH = 1920
 BROWSER_HEIGHT = 1080
```
Comment (mattmahoneyrh) [inline conf/properties.properties]: properties.properties should not have any modifications - please revert to current properties.properties in 'master'

**Comment by mattmahoneyrh** [comment]
This PR is no longer relevant.

**Pull request closed without merging**